assets/javascripts/test_result.js

```diff
@@ -62,7 +62,7 @@ const tabConfiguration = {
   investigation: {
     descriptiveName: 'investigation info',
     conditionForShowingNavItem: function () {
-      return testStatus.state === 'done' && testStatus.result === 'failed';
+      return testStatus.state === 'done' && (testStatus.result === 'failed' || testStatus.result === 'incomplete');
```

Comment:
Would be nice if we could also make this "if we have inestigation data" rather than adding more states. But I guess we don't really know that here?

Reply:
I suppose we can always run the investigation. It is just a matter how useful it'll be. I would avoid running the investigation at the time we decide which tabs to show because running the investigation isn't for free.